lib/PrimitiveHelpers.js

```diff
@@ -185,6 +216,13 @@ function getPrimitivesByMaterialMode(primitives) {
     return primitivesByMaterialMode;
 }
 
+/**
+ * Process the glTF asset primitives and return primitives that share attribute accessors.
+ *
+ * @param {Array} primitives An array of primitive objects to compare the primitve against.
+ * @param {Object} primitive The primitve to compare for conflicts.
+ * @returns {Array} An array conmtaining all the primitives that share attribute accessor.
```

Comment:
{Number[]}

"An array containing indexes of primitives that have attribute accessor conflicts"